figures/monolith-id-sequence.mmd

```diff
@@ -0,0 +1,9 @@
+sequenceDiagram
+    participant Monolith
+    participant Balancer
+
+    Monolith->>Monolith: Generate Monolith ID on startup
+    Monolith->>Balancer: Send Monolith ID in Init Message
+    Balancer->>Monolith: Connects Balancer to Monolith
```

Comment:
These are backwards.

Reply:
and the caption can just be "Connect"

Reply:
Thought they should be in that order since balancer connects to monolith. Changing now.